### DISCUSSION TITLE
Text and Table Same Time

### DISCUSSION THREAD

**Discussion by chozillla** [jsvine/pdfplumber#1183] [Q&A] [answered]
Hi, 

Is there a way to both get text and the text from tables( keeping the structure and info in the table) at the same time?

**Comment by jsvine** [comment, marked as answer]
Hi @chozillla, and thanks for your interest in `pdfplumber`. There is not a built-in function for this, but you can find some related discussions (and approaches to achieving your goal) here:

- https://github.com/jsvine/pdfplumber/discussions/1026
- https://github.com/jsvine/pdfplumber/discussions/1012
- https://github.com/jsvine/pdfplumber/discussions/1005
